Makefile

```diff
@@ -306,15 +307,21 @@ docker-images-jaeger-backend-debug: SUFFIX = -debug
 
 .PHONY: docker-images-jaeger-backend docker-images-jaeger-backend-debug
 docker-images-jaeger-backend docker-images-jaeger-backend-debug: create-baseimg create-debugimg
-	for component in agent collector query ingester ; do \
-		docker build --target $(TARGET) \
-			--tag $(DOCKER_NAMESPACE)/jaeger-$$component$(SUFFIX):${DOCKER_TAG} \
+	for component in "jaeger-agent" "jaeger-collector" "jaeger-query" "jaeger-ingester" "all-in-one" ; do \
+		regex="jaeger-(.*)"; \
+		component_suffix=$$component; \
+		if [[ $$component =~ $$regex ]]; then \
+			component_suffix="$${BASH_REMATCH[1]}"; \
+		fi; \
+		docker buildx build --target $(TARGET) \
+			--tag $(DOCKER_NAMESPACE)/$$component$(SUFFIX):${DOCKER_TAG} \
 			--build-arg base_image=$(BASE_IMAGE) \
 			--build-arg debug_image=$(DEBUG_IMAGE) \
 			--build-arg TARGETARCH=$(GOARCH) \
-			cmd/$$component ; \
+			--load \
+			cmd/$$component_suffix; \
 		echo "Finished building $$component ==============" ; \
-	done
+	done; \
```

Comment:
why does this have `\` continuation?

Reply:
It was a remnant of the previous implementation where the IFS needed to be reset. Removed in [c649fa6](https://github.com/jaegertracing/jaeger/pull/3813/commits/c649fa6732d0660ef020bc081019ae41931e4986)